apache/docker-entrypoint.sh

```diff
@@ -29,6 +29,45 @@ if [ ! -z "${PMA_USER_CONFIG_BASE64}" ]; then
     echo "${PMA_USER_CONFIG_BASE64}" | base64 -d > /etc/phpmyadmin/config.user.inc.php
 fi
 
+if [ ! -z "${PMA_SSL_CA_BASE64}" ]; then
+    mkdir -p /etc/phpmyadmin/ssl
+    echo "Adding the custom pma-ssl-ca from base64."
+    echo "${PMA_SSL_CA_BASE64}" | base64 -d > /etc/phpmyadmin/ssl/pma-ssl-ca.pem
+    export "PMA_SSL_CA"="/etc/phpmyadmin/ssl/pma-ssl-ca.pem"
+fi
+
+if [ ! -z "${PMA_SSL_KEY_BASE64}" ]; then
+    mkdir -p /etc/phpmyadmin/ssl
+    echo "Adding the custom pma-ssl-key from base64."
+    echo "${PMA_SSL_KEY_BASE64}" | base64 -d > /etc/phpmyadmin/ssl/pma-ssl-key.key
+    export "PMA_SSL_KEY"="/etc/phpmyadmin/ssl/pma-ssl-key.key"
+fi
+
+if [ ! -z "${PMA_SSL_CERT_BASE64}" ]; then
+    mkdir -p /etc/phpmyadmin/ssl
+    echo "Adding the custom pma-ssl-cert from base64."
+    echo "${PMA_SSL_CERT_BASE64}" | base64 -d > /etc/phpmyadmin/ssl/pma-ssl-cert.pem
+    export "PMA_SSL_CERT"="/etc/phpmyadmin/ssl/pma-ssl-cert.pem"
+fi
+
+if [ ! -z "${PMA_SSL_CAS_BASE64}" ]; then
+    echo "Adding multiples custom pma-ssl-ca from base64."
+    PMA_SSL_CAS=$(generate_ssl_files "${PMA_SSL_CAS_BASE64}" "CA" "pem")
+    export "PMA_SSL_CAS"
+fi
+
+if [ ! -z "${PMA_SSL_KEYS_BASE64}" ]; then
+    echo "Adding multiples custom pma-ssl-key from base64."
+    PMA_SSL_KEYS=$(generate_ssl_files "${PMA_SSL_KEYS_BASE64}" "CERT" "cert")
+    export "PMA_SSL_KEYS"
+fi
+
+if [ ! -z "${PMA_SSL_CERTS_BASE64}" ]; then
+    echo "Adding multiples custom pma-ssl-cert from base64."
+    PMA_SSL_CERTS=$(generate_ssl_files "${PMA_SSL_CERTS_BASE64}" "KEY" "key")
+    export "PMA_SSL_CERTS"
+fi
+
```

Comment:
I am not too much fan of this part of the PR, could we do it in the php config file ?

Reply:
Modifications are done. I moved the TLS logic inside the PHP configuration file. I have also added a helpers file to move the renamed decodeAndSaveSslFiles function out of the configuration. This way, we will be able to add more helpers if needed.

Tests have been done with my production environment, which requires mutual TLS. Files have been generated as expected, and secure communication is working as well:

![image](https://github.com/user-attachments/assets/e9af9c67-4d0e-41f5-9bc8-0b6773cf5377)

![image](https://github.com/user-attachments/assets/295c2957-8593-4888-9534-e09d4996d390)